src/payment/strategies/affirm/affirm-payment-strategy.ts

```diff
@@ -126,17 +126,17 @@ export default class AffirmPaymentStrategy implements PaymentStrategy {
             shipping: this._getShippingAddress(),
             billing: this._getBillingAddress(),
             items: this._getItems(),
-            discounts: this._getDiscounts(),
             metadata: {
                 shipping_type: consignment.selectedShippingOption.type,
                 mode: 'modal',
+                platform_type: 'bigcommerce',
```

Comment:
platform_type value must be BigCommerce, the other two values related to the platform must be kept in blank